docs/addons/addon-types.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Types of addons'
+---
+
+Each Storybook addon is classified into two general categories, UI-based or Presets. Documented here are some of the features available for each type of addon. Use it as a reference when creating your next Storybook addon.
+
+## UI-based addons
+
+This particular type of addons allows you to customize Storybook's UI. If the addon you're developing falls into this category you can use some of the following elements to further customize the interface.
```

Comment:
```suggestion
UI-based addons allow you to customize Storybook's UI using the following elements.
```

Edit for conciseness. This can be shortened into one sentence. 

Also be judicious about "some", that word is imprecise and doesn't give the reader confidence in the docs.